### ISSUE TITLE
Outdated WalletLink Code --> Needs to be updated to Coinbase Wallet SDK

### ISSUE THREAD

**Issue by TrevvvD** [amaryfilo/connect-wallet#8]
Hi,

I work on the DevRel team at Coinbase and am submitting this issue to inform you that the WalletLink SDK has been deprecated in favor of our new [Coinbase Wallet SDK](https://github.com/coinbase/coinbase-wallet-sdk). You can find more on that and how to upgrade to the latest version [here](https://docs.cloud.coinbase.com/wallet-sdk/docs/welcome).

Please let me know if you have any questions!

Thanks,
Trevor

**Comment by Twenx1** [comment]
Hello! I was just following up on what my friend TrevvvorD was stating up top here. If there are any questions we could answer, please let us know. We are super down to help.

Thanks,
Freddy

**Comment by amaryfilo** [comment]
**Hello**! 
Thank you for this Issue! @freddt @TrevvvD
Now I've started working on it 🔭

I found some bugs 🐞 when using MM and CW together. When I fix this, I will create a pre-release.

**Comment by amaryfilo** [comment]
@FreddyMJ3 I can’t create transactions in Eth Testnet, because the “Developer mode” button is “hidden” (I could not see button in the extension because there was no scrolling page, sometimes it is absent :c). 

Perhaps this is a cool function to close only Ethereum Testnet, but I do not understand why this is needed. 🤔 

I added another Testnet blockchain (not ETH), and everything works fine! But not with Kovan and Rinkeby, because users must turn on the developer mode to create a test transaction to see, for example, “how this application works”.

**Comment by amaryfilo** [comment]
Pre-release version now available  [1.1.25-dev.98a738b0b](https://www.npmjs.com/package/@amfi/connect-wallet/v/1.1.25-dev.98a738b0b)

**Install via**
```console
npm install -D @amfi/connect-wallet@1.1.25-dev.98a738b0b
```

**Comment by amaryfilo** [comment]
**Release** [v1.1.25](https://github.com/amaryfilo/connect-wallet/releases/tag/v1.1.25)

Install via npm or yarn.
```console
npm install -D @amfi/connect-wallet@1.1.25
```
```console
yarn add -D @amfi/connect-wallet@1.1.25
```
